layouts/docs/changelog.html

```diff
@@ -0,0 +1,46 @@
+{{- /*
+
+    Template to render a page with a `changelog` layout or the `changelog`
+    section page. This conflation seems to be a limitation of Hugo currently, it
```

Comment:
Note: this conflation could be avoided by using a different name for the layout, but on the other end I don't think it hurts that much and avoids to have a shortcode just for redirecting.